### PR TITLE
feat(articles): add displayArticle method

### PR DESCRIPTION
**Pull request by Oupsla**

### CODE REVIEW

File: src/web.ts [hunk 1]
```diff
@@ -120,6 +121,11 @@ export class IntercomWeb extends WebPlugin implements IntercomPlugin {
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async displayArticle(_article: IntercomArticle): Promise<void> {
+    throw new Error('[Intercom] displayArticle is not implemented on web');
+  }
+
```
Comment (piitaya) [inline src/web.ts:128]: Maybe a no-op with a log error ? Or open the article in a new tab using faq link?

**Review by piitaya — APPROVED** [review]

File: src/web.ts (hunk 1 above)
Comment (Oupsla) [reply src/web.ts:128]: IMO, better a noop (but throwing an error is better no ?)
And the handling of opening the URL in a new tab can be handled by the app directly

Comment (piitaya) [reply src/web.ts:128]: Yes, it make sense 🙂

Comment (piitaya) [reply src/web.ts:128]: Maybe you can use this : https://github.com/ionic-team/capacitor/blob/main/core/src/web-plugin.ts#L93-L99

**Pull request merged**